manual/parts/threads.tex

```diff
@@ -54,9 +54,11 @@ \subsection{Thread Creation}
 scheduling context object.
 
 In a SMP configuration of the kernel, the thread will resume on the core
-corresponding to the affinity of the thread. For the standard (non-MCS) kernel, this is set using
-\apifunc{seL4\_TCB\_SetAffinity}{tcb_setaffinity}, while on the MCS kernel the affinity is
-derived from the scheduling context object.
+corresponding to its affinity. For the standard (non-MCS) kernel, this is set
+using \apifunc{seL4\_TCB\_SetAffinity}{tcb_setaffinity}. By default, a thread's
+affinity is the core where the thread has been running on, that created the
+\obj{TCB} object. On the MCS kernel, the affinity is derived from the
+scheduling context object.
```

Comment:
Agreed that this change is an improvement.

Perhaps clearer: "By default, a thread's affinity is the core where its \obj{TCB} object was created."

Reply:
done